guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -4,8 +4,42 @@
 [id="katello-headline-features"]
 == Headline Features
 
+=== Verify Content Checksum on container repositories using Hammer
+Added option '--validate-contents true/false' to "hammer repository synchronize" command.
+
+=== Add ACS bulk refresh option
+Added option '--all' to "hammer alternate-content-source refresh" command.
+ACS bulk refresh through API silently sanitizes input ids.
+
+=== Experimental activation key details page
+When the setting 'Show Experimental Labs' is enabled, the Activation Key 
+details page will be replaced by the beginnings of a redesigned page.
+
+=== Content Views and CVV enhancements to filters and refresh
+Content View filter creation/edit button now says 'Save' instead of 'Edit Rule'.
+Content view dependency solving tracked for needs_publish.
+
+=== Pulpcore and plugins updated from Pulpcore 3.22 to 3.28
+Contains both fixes and features that Katello can use in the future to better manage content, such as:
+https://github.com/pulp/pulp_rpm/issues/3135[More filter options on the packages API]
+https://github.com/pulp/pulp_rpm/issues/3009[New setting for generated primary.xml timestamps to match upstream repository]
+https://github.com/pulp/pulp_rpm/issues/2995[RPM replicator support]
+https://github.com/pulp/pulpcore/issues/3538[New distribution hiding option]
+https://github.com/pulp/pulpcore/issues/3835[OpenTelemetry support]
+https://github.com/pulp/pulpcore/issues/3656[File size added to content application listing]
+
 [id="katello-upgrade-warnings"]
 == Upgrade Warnings
 
-[id="katello-deprecations"]
-== Deprecations
+=== Katello-Agent removal
+Katello-Agent has been removed from Katello and is no longer available via web UI, API, or CLI.
+katello-agent will be removed by upgrading.
+For patching hosts going forward, use remote execution.
+Remote execution will not be automatically enabled as part of the upgrade. 
+To ensure uninterrupted host patching, migrate to remote execution before upgrading.
+Guide for migrating to remote execution:
+[https://github.com/theforeman/foreman-documentation/blob/3.8/guides/common/modules/proc_migrating-hosts-from-katello-agent-to-remote-execution.adoc].
+
+=== Remove the setting 'Use remote execution by default'
```

Comment:
The heading says one setting is removed, but the body mentions 3 settings.

As for a suggestion I'm debating on a simple heading like:
```suggestion
=== Removed various settings
```

Or since it's related to the agent removal you can also squash it into the previous section.